assets/scss/components/_dashboard.scss

```diff
@@ -0,0 +1,81 @@
+.dashboard__main {
```

Comment:
This could be more nested like below, but I find this more readable given the complexity of the `dashboard__card` styles

```
.dashboard__ {
  &main {
  }
  
  &card {
    &--linked {
    }

    &-wrapper {
    }

    &-heading {
    }

    &-description {
    }

    &s {
      &-wrapper {
      }
    }
  }
}
```